boards/px4/fmu-v4/default.px4board

```diff
@@ -5,8 +5,7 @@ CONFIG_BOARD_SERIAL_GPS1="/dev/ttyS3"
 CONFIG_BOARD_SERIAL_TEL1="/dev/ttyS1"
 CONFIG_BOARD_SERIAL_TEL2="/dev/ttyS2"
 CONFIG_BOARD_SERIAL_WIFI="/dev/ttyS0"
-CONFIG_DRIVERS_ADC_ADS1115=y
-CONFIG_DRIVERS_ADC_BOARD_ADC=y
+CONFIG_DRIVERS_ADC_ADS7953=y
```

Comment:
please don't remove the existing entries

Reply:
I have added the entries and pushed the commit. Please check.